tests/getAddLiquidityExtrinsic.test.ts

```diff
@@ -0,0 +1,46 @@
+import getAddLiquidityExtrinsic from "@/utils/getAddLiquidityExtrinsic";
+import { Balance, fetchPoolExchangeInfo } from "@/utils";
+
+const api = global.getCENNZApiForTest();
+const { cennzAsset, cpayAsset } = getCENNZCoreAssetsForTest();
+
+describe("getAddLiquidityExtrinsic", () => {
+	it("returns expected extrinsic", async () => {
+		const exchangeInfo = await fetchPoolExchangeInfo(
+			api,
+			cennzAsset,
+			cpayAsset
+		);
+
+		const coreAssetValue = new Balance(10, cpayAsset);
+		const tradeAssetValue = new Balance(10, cennzAsset);
+		const slippage = 5;
+
+		const extrinsic = await getAddLiquidityExtrinsic(
+			api,
+			exchangeInfo,
+			cennzAsset.assetId,
+			tradeAssetValue,
+			coreAssetValue,
+			slippage
+		);
+
```

Comment:
the more effective way to test the extrinsic is to inspect its `.args` property and ensure the variables are constructed as expected

![Screen Shot 2022-04-29 at 13 31 58](https://user-images.githubusercontent.com/97480229/165872615-539b516a-22e5-4376-9c5d-e01e26a94001.png)

Reply:
not sure I can, all I've got is

```
console.log
    extrinsic Submittable {
      createdAtHash: undefined,
      initialU8aLength: undefined,
      signaturePayloadOptions: {
        blockHash: undefined,
        genesisHash: undefined,
        method: undefined,
        specVersion: undefined,
        nonce: undefined,
        runtimeVersion: undefined,
        transactionPayment: undefined,
        transactionVersion: undefined
      }
    }
```